certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/configuration/MessagingConfig.java

```diff
@@ -0,0 +1,49 @@
+package se.inera.intyg.certificateservice.infrastructure.configuration;
+
+import jakarta.jms.ConnectionFactory;
+import jakarta.jms.Queue;
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.apache.activemq.command.ActiveMQQueue;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.core.JmsTemplate;
+
+@Configuration
+public class MessagingConfig {
+
+  @Value("${certificate.service.amq.name}")
+  private String eventQueueName;
+
+  @Value("${spring.activemq.broker-url}")
+  private String brokerUrl;
+
+  @Value("${spring.activemq.user}")
+  private String amqUser;
+
+  @Value("${spring.activemq.password}")
+  private String amqPassword;
+
+  @Bean
+  public Queue eventQueue() {
+    return new ActiveMQQueue(eventQueueName);
+  }
+
+  @Bean
+  public ConnectionFactory jmsConnectionFactory() {
+    return new ActiveMQConnectionFactory(amqUser, amqPassword, brokerUrl);
+  }
```

Comment:
Check if this should be a PoolingConnectionFactory. Perhaps have a look at Intygstjanst.

Reply:
I saw this in a guide: PooledConnectionFactory is not JMS 2.0 compliant